lib/rubocop/cop/lint/debugger.rb

```diff
@@ -44,6 +44,7 @@ class Debugger < Cop
 
         def_node_matcher :debugger_call?, <<~PATTERN
           {(send {nil? #kernel?} {:debugger :byebug} ...)
+           (send {nil? #kernel?} {:debugger :remote_byebug} ...)
```

Comment:
Please update the node pattern because `:debugger` is duplicated.

```diff
-{(send {nil? #kernel?} {:debugger :byebug} ...)
- (send {nil? #kernel?} {:debugger :remote_byebug} ...)
+{(send {nil? #kernel?} {:debugger :byebug :remote_byebug} ...)
```

cf . https://github.com/rubocop-hq/rubocop/blob/v0.74.0/lib/rubocop/node_pattern.rb#L35